packages/cli/src/cli/cogify/action.make.cog.ts

```diff
@@ -86,6 +87,12 @@ export class CommandMakeCog extends CommandLineAction {
       description: 'Aligned level between resolution and cog',
       required: false,
     });
+    this.concurrency = this.defineIntegerParameter({
+      argumentName: 'CONCURRENCY',
+      parameterLongName: '--concurrency',
+      description: 'Number of concurrency create cog jobs',
+      required: false,
+    });
```

Comment:
Would it be better to set the default concurrency value here via `defaultValue`, rather than in a constant on line 24? That would mean we wouldn't need to check if it had been supplied every time it is used.

Reply:
Very good point. This default value won't be set into `this.concurrency.value` if this parameter not used, the use case is `this.concurrency.defaultValue`, so we still need to check exists and set `this.concurrency.defaultValue` in not exists.

Reply:
> This default value won't be set into this.concurrency.value if this parameter not used, the use case is this.concurrency.defaultValue

That is just a typing problem, and one of the reasons I think we should change our cli parser.

if no value is provided, the `this.concurrency.value` will be `defaultValue`